### PR TITLE
CMSIS-NN: Update softmax, svdf and pooling unit tests

### PR DESCRIPTION
**Pull request by mansnils**
Data generate script uses TFL int kernels instead of TF float kernels.
Softmax and pooling test cases use Keras for generating a model.
Svdf use a json template for generating a model.
Corresponding input headers are regenerated.

### CODE REVIEW

**Comment by grasci-arm** [comment]
Can one of the admins verify this patch?
